package/MDAnalysis/analysis/contacts.py

```diff
@@ -14,45 +14,35 @@
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
 
-"""
-Native contacts analysis --- :mod:`MDAnalysis.analysis.contacts`
+"""Native contacts analysis --- :mod:`MDAnalysis.analysis.contacts`
 ================================================================
 
-Analysis of native contacts *q* over a trajectory.
-
-* a "contact" exists between two atoms *i* and *j* if the distance between them is
-  smaller than a given *radius*
-
-* a "native contact" exists between *i* and *j* if a contact exists and if the
-  contact also exists between the equivalent atoms in a reference structure or
-  conformation
 
-The "fraction of native contacts" *q(t)* is a number between 0 and 1 and
-calculated as the total number of native contacts for a given time frame
-divided by the total number of contacts in the reference structure.
+Analysis of native contacts *Q* over a trajectory. Native contacts of a
+conformation are contacts that exist in a reference structure and in the
+conformation. Contacts in the reference structure are always defined as being
+closer then a distance `radius`. The fraction of native contacts for a
+conformation can be calculated in different ways. This module supports 3
+different ones
```

Comment:
maybe something like "This module supports 3 different metrics listed below, as well as custom metrics", then you can remove lines 43-45 as that information is now here